src/value.h

```diff
@@ -41,8 +47,8 @@ typedef enum {
 
 #define SI_TYPE(value) (value).type
 #define SI_NUMERIC (T_INT64 | T_DOUBLE)
-#define SI_STRING (T_STRING | T_CONSTSTRING)
 #define SI_GRAPHENTITY (T_NODE | T_EDGE)
+#define SI_ALL (T_MAP | T_NODE |T_EDGE | T_ARRAY | T_PATH | T_DATETIME | T_LOCALDATETIME | T_DATE | T_TIME | T_LOCALTIME | T_DURATION | T_STRING | T_BOOL | T_INT64 | T_DOUBLE | T_NULL | T_PTR)
```

Comment:
what is the usage of SI_ALL?

Reply:
In the `runtime-arithmetic-error` branch, we use combined SITypes to register what values a function can accept, so it's used by some of those functions (especially comparisons).